src/containers/unicode_point_map.rs

```diff
@@ -0,0 +1,1124 @@
+// containers/unicode_point_map.rs : `UnicodePointMap`
+
+use base_traits::{
+    IsEmpty,
+    Len,
+};
+
+use std::{
+    collections::HashMap,
+    ops as std_ops,
+};
+
+
+/// Constants
+pub(crate) mod constants {
+
+    #[cfg(debug_assertions)]
+    pub(crate) const DEFAULT_CONTIGUOUS_CEILING : char = '\u{80}';
+    #[cfg(not(debug_assertions))]
+    pub(crate) const DEFAULT_CONTIGUOUS_CEILING : char = '\u{1000}';
+}
+
+
+/// A container that measures the frequences of the unique characters that
```

Comment:
Documentation should be explicit that this is for code points.
```suggestion
/// A container that measures the frequencies of the unique code points that
```